// Copyright 2025 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.
//
// Sync protocol datatype extension for themes on iOS.

// If you change or add any fields in this file, update proto_visitors.h and
// potentially proto_enum_conversions.{h, cc}.

syntax = "proto2";

option java_multiple_files = true;
option java_package = "org.chromium.components.sync.protocol";

option optimize_for = LITE_RUNTIME;

package sync_pb;

import "components/sync/protocol/theme_types.proto";

// Properties of theme sync objects for iOS themes.
message ThemeSpecificsIos {
  optional UserColorTheme user_color_theme = 1;

  optional NtpCustomBackground ntp_background = 2;
}
